style.css

```diff
@@ -529,3 +480,250 @@ nav #mylogo {
 .bottonIcon > img {
   width: calc(100% / 2);
 }
+
+.drop-li {
+  display: flex;
+  background-color: #f7f7f9;
+  gap: 2%;
+  margin-top: 12px;
+}
+
+@media (max-width: 279px) {
+  .drop-li {
+    flex-direction: column;
+  }
+}
+
+@media (max-width: 325px) {
+  .about-text {
+    font-weight: 400;
+    line-height: 24px;
+  }
+}
+
+@media (min-width: 504px) {
+  .drop-box {
+    right: 2%;
+  }
+}
+
+@media (min-width: 507px) {
+  #Headline-Text {
+    width: 90%;
+  }
+
+  .Navbar {
+    justify-content: space-between;
+  }
+}
+
+@media (min-width: 672px) {
+  .drop-box {
+    right: 0;
+  }
+}
+
+@media (min-width: 768px) {
+  .Box {
+    flex-direction: row;
+    width: 90%;
+    margin-left: 5%;
+  }
+
+  .contact-form {
+    margin: 50px auto;
+    width: 50%;
+  }
+
+  .contact-form > input,
+  .contact-form > textarea {
+    width: 100%;
+  }
```

Comment:
- [ ] Please notice the get in touch button is centered in the original design, yours is aligned to the left please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/221257410-21e812d6-3631-4d4b-9169-61c4c6fe01f6.png)

how it should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/221257520-fdf7bd4d-80f9-46f9-bca5-5ab0d3c21fad.png)